### PR TITLE
Fixing 404s in Bookinfo demo and consul install files.

### PR DESCRIPTION
**Pull request by rshriram**
Set the default refresh delays back to 1s, so that the bookinfo demo works as it used to. We can add docs telling users to set the delays to higher values in general (provided they can withstand the brief 404s). Otherwise, they will have to take the 30-40% perf hit (for just ingress alone) caused by frequent refresh until we move to v2 APIs. [https://github.com/istio/pilot/issues/1237]

Also fix consul control plane install files (move them to install/consul/..). Closing the other PR #955 as both passed e2e, and this PR is superset of the other one.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by rshriram** [comment]
This is stacked on top of the Consul PR. And the diffs will go away once that is merged.

File: install/kubernetes/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Install Istio on an existing Kubernetes cluster
 
-Please follow the installation instructions from [istio.io](https://istio.io/docs/tasks/installing-istio.html).
+Please follow the installation instructions from [istio.io](https://istio.io/docs/setup/kubernetes/).
```
Comment (ldemailly) [inline install/kubernetes/README.md]: why not link directly into https://istio.io/docs/setup/kubernetes/quick-start.html ?

**Comment by ldemailly** [comment]
Any explanation as to why changing from 30s to 1s 100% eliminates the 404 issue ?
(would 29s do the same if the other rate is 30s or if not wouldn't it just reduce by 29/30th but not eliminate the race/issue)

**Comment by kyessenov** [comment]
@ldemailly it doesn't but it makes less noticeable.
When you change a rule, envoy needs to fetch two pieces of update within the poll period (1-2x of 30s or 1s after this PR). If the two updates come out of order, you get 404.

**Comment by ldemailly** [comment]
@kyessenov thanks, so it shortens the window for 404s from 30s to 1s

any chance we can be creative about fixing the root issue instead ?

**Comment by kyessenov** [comment]
The root issue is in C++ code in envoy. We can be creative in pilot, but that might introduce stability risk, so I'd rather do it right next time.

**Comment by ldemailly** [comment]
tickling the bots

**Pull request merged**